.github/workflows/ci.yml

```
@@ -431,6 +431,25 @@ jobs:
       - name: Run dependency check
         run: ./ci/check_job_dependencies.sh
 
+  test-packing:
```

When we use the word "packing" in the case of zerocopy, we're usually talking about uninitialized bytes added to layouts to satisfy alignment requirements. I think "packaging" is less prone to be confused with that. We also start our job names with `check-`.
```suggestion
  check-packaging:
```